Prog/mt_vern_profile_data.sas

```diff
@@ -378,12 +384,12 @@ data compile_mvt_tabs_tr10_select;
 	set compile_mvt_tabs_tr10;
 	if Geo2010 in("11001004701","11001004702") then target = 1;
 	if Geo2010 in("11001010600","11001004600","11001004802","11001004902","11001005800","11001005900") then target = 2;
-if Geo2010 not in("11001004701","11001004702","11001010600","11001004600","11001004802","11001004902","11001005800","11001005900") then delete;
+	if Geo2010 not in("11001004701","11001004702","11001010600","11001004600","11001004802","11001004902","11001005800","11001005900") then delete;
 run;
 
 data compile_mvt_tabs_wd12_select;
 	set compile_mvt_tabs_wd12;
-	if Ward2012 in("1","2","3","4","5") then delete;
+	if Ward2012 in("3","4","5","7","8") then delete;
 	geography = Ward2012;
```

Comment:
should specify a length statement for the geography var - its getting truncated for one of them.